bin/oref0-cron-nightly.sh

```diff
@@ -17,5 +17,5 @@ directory="$PWD"
 
 if [[ $ENABLE =~ autotune ]]; then
     # autotune nightly at 4:05am using data from NS
-    (oref0-autotune -d=$directory -n=$NIGHTSCOUT_HOST && cat $directory/autotune/profile.json | jq . | grep -q start && cp $directory/autotune/profile.json $directory/settings/autotune.json) 2>&1 | tee -a /var/log/openaps/autotune.log &
+    (oref0-autotune -d=$directory -n=$NIGHTSCOUT_HOST && cat $directory/autotune/profile.json | jq . | grep -q start && cp $directory/autotune/profile.json $directory/settings/autotune.json && oref0-upload-profile settings/profile.json $NIGHTSCOUT_HOST $API_SECRET) 2>&1 | tee -a /var/log/openaps/autotune.log &
```

Comment:
I think you would want to upload either the `$directory/autotune/profile.json` or the `$directory/settings/autotune.json` rather than the `$directory/settings/profile.json` because it won't be updated yet to the new basal profile in the `$directory/settings/autotune.json` by the time it runs. 

The downside is that you won't pick up changes made in the preferences.json file over time.  There is a separate issue opened up on autotune not picking up those changes.  Once that is resolved, I don't think `$directory/settings/autotune.json`  will have a downside.